### PR TITLE
Fix CI by ensuring `gotestfmt` is on PATH, plus cleanups

### PR DESCRIPTION
**Pull request by DMRobertson**
See commits for details.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -53,27 +48,20 @@ jobs:
...
       # to do this so we can _be_ the host when running Complement so we can snaffle all the ports. If
       # we run Complement _in_ Docker then we can't -p all high numbered ports which then breaks federation
       # servers which listen on random high numbered ports.
       - name: "Install Complement Dependencies"
-        # We don't need to install Go because it is included on the Ubuntu 20.04 image:
-        # See https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md specifically GOROOT_1_17_X64
         run: |
           sudo apt-get update && sudo apt-get install -y libolm3 libolm-dev
-          go get -v github.com/gotesttools/gotestfmt/v2/cmd/gotestfmt@latest
+          go install -v github.com/gotesttools/gotestfmt/v2/cmd/gotestfmt@latest
```
Comment (clokep) [inline .github/workflows/ci.yaml:61]: Probably a version of go changing stuff?

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -53,27 +48,20 @@ jobs:
...
           mkdir .gotestfmt/github -p
           cp .ci/complement_package.gotpl .gotestfmt/github/package.gotpl
+          gotestfmt -help
```
Comment (clokep) [inline .github/workflows/ci.yaml:64]: This is just to fail fast if it doesn't exist?

**Review by clokep — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (DMRobertson) [reply .github/workflows/ci.yaml:64]: Exactly!

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (DMRobertson) [reply .github/workflows/ci.yaml:61]: I think so. I didn't dig too hard here!

**Pull request merged**